apps/full-site-editing/full-site-editing-plugin/homepage-posts/editor.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ * Internal dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+/* eslint-enable import/no-extraneous-dependencies */
+
+/**
+ * NHA dependencies
+ */
+import { settings } from './newspack-homepage-articles/blocks/homepage-articles/index';
+
+registerBlockType( 'a8c/homepage-posts', {
```

Comment:
Should we ensure set up the `title` beyond that we've already changed it in the Newspack side?
In this way, the name will keep as we want. Also, I think we should rename the name changing the namespace/prefix to `a8c`.

```es6
registerBlockType( 'a8c/homepage-posts', {
 	...settings,
    title: __( 'Homepage posts' ),
    name: 'a8c/homepage-posts',
	category: 'layout',
} );
```

Reply:
For now, I have removed the specific redeclaration you mention to avoid unnecessary overwrites. The block will likely get a new name for dotcom soon and once it is known, let's add it back 👍 paYJgx-li-p2